src/widget/widget.cpp

```diff
@@ -265,7 +267,7 @@ void Widget::init()
     connect(setStatusAway, SIGNAL(triggered()), this, SLOT(setStatusAway()));
     connect(setStatusBusy, SIGNAL(triggered()), this, SLOT(setStatusBusy()));
     connect(addFriendForm, SIGNAL(friendRequested(QString, QString)), this, SIGNAL(friendRequested(QString, QString)));
-    connect(idleTimer, &QTimer::timeout, this, &Widget::onUserAwayCheck);
+    connect(timer, &QTimer::timeout, this, &Widget::onTick);
```

Comment:
Wouldn't it be better just to have on function for each thing using the timer, and then have one connect for each function?

As opposed to having a singular onTick() that does a billion things. (It's only two small things now, but if this is meant to be a general purpose timer then onTick wouldn't scale well.